sdk/src/test/java/com/entracer/sdk/analytics/EntracerAPITest.java

```diff
@@ -0,0 +1,104 @@
+package com.entracer.sdk.analytics;
+
+import com.entracer.sdk.model.Organisation;
+import com.entracer.sdk.model.Person;
+import com.entracer.sdk.network.Request;
+import com.entracer.sdk.network.ResponseListener;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
```

Comment:
You may change this comment as well, meantime please add a meaningful git commit message for commit like `Reformat code` or something